pom.xml

```diff
@@ -112,8 +112,8 @@
       <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
       <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
 
-      <maven.compiler.source>1.7</maven.compiler.source>
-      <maven.compiler.target>1.7</maven.compiler.target>
+      <maven.compiler.source>1.8</maven.compiler.source>
+      <maven.compiler.target>1.8</maven.compiler.target>
```

Comment:
That's what I meant. BUT, https://activemq.apache.org/artemis/docs/1.3.0/preface.html says
> Written in Java. Runs on any platform with a Java 8+ runtime, that's everything from Windows desktops to IBM mainframes.

That would mean people on 1.7 are out.